src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -212,7 +212,7 @@ case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
               A_matx(1, ::) := n2.t
               A_matx(2, ::) := n3.t
               val p_vect = A_matx \ b_vect
-              Some((p_vect(0), p_vect(1), p_vect(2)))
+              Some((p_vect(0) + centerX, p_vect(1) + centerY, p_vect(2) + centerZ))
```

Comment:
Originally, the vertices we're being calculated in a local coordinate system. Then the centroid was calculated in the local coordinate system and translated into the global coordinates. 
What it's doing now is calculating the vertices in the global coordinate system. When the centroid is calculated later, there's no need to translate it since it's being calculated from global coordinates. 
The vertices are needed in global coordinates for the vtk file, so that's why I did this. Does that make sense?